packages/frontend/src/components/otherSites/OtherSites.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+
+import { OutLink } from '../OutLink'
+
+export function OtherSites() {
+  return (
+    <div
+      className="
+      mt-[32px] flex flex-row items-center
+      justify-between rounded-[12px] bg-gradient-to-r
+      from-[#7E41CC99] 
```

Comment:
avoid inlining colors, add them to tailwind config instead

Reply:
yup, and if there is really similar color then use this one instead